marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/provenance/data-hub-provenance/test-data/steps/ingestion/loadCustomersJSON.step.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
If this works, that's kind of okay... but it doesn't make sense to test an ingestion step from ML code since in order to run an ingestion step, you have to use the DHF Java code (WriteStepRunner and FlowRunner). So this test raises questions of - wait, what happens if you run an ingestion step from ML code? And is the result of that even valid for testing? 

I recommend instead using the createSimpleProject or createSimpleMappingProject function in data-hub-test-helper.sjs instead. Then you don't need to create a flow file or a step file. Since you don't care about what the mapping actually does, you don't need to pass anything into createSimpleMappingProject.